dynamic-demo-plugin/i18next-parser.config.js

```diff
@@ -4,5 +4,9 @@ module.exports = {
   locales: ['en'],
   namespaceSeparator: '~',
   reactNamespace: false,
-  useKeysAsDefaultValue: true,
+  defaultValue: function (_, _, key, _) {
+    // The `useKeysAsDefaultValues` option is deprecated in favor of `defaultValue` option function arguments.
+    // The `key` is used to set default value.
+    return key;
+  }
```

Comment:
Why is this needed here but not in the core app?

Reply:
> Why is this needed here but not in the core app?

@TheRealJon The core app make use of the i18next-parser with a custom [script](https://github.com/openshift/console/blob/master/frontend/i18n-scripts/set-english-defaults.js) for parsing the strings. If we update the core app `defaultValue` with the function arguments, it would we'll have return empty string - `''` not `key` since the custom script handle that and more parser functions. So it is cleaner to assign `''` to `defaultValue` in core app.